SPECS/memtest86.spec

```
@@ -0,0 +1,35 @@
+# Debug package is empty and rpmlint rejects build
+%define _enable_debug_packages %{nil}
+%define debug_package %{nil}
+
+Summary:	A stand alone memory test for x86 architecture systems
+Name:		memtest86+
+Version:	6.01
+Release:	1%{?dist}
+License:	GPLv2
+Group:		System/Kernel and hardware
+Url:		http://www.memtest.org
+Source0:	https://github.com/memtest86plus/memtest86plus/archive/refs/tags/v%{version}.tar.gz
+BuildRequires:	gcc binutils
+ExclusiveArch:	x86_64
+
+%description
+MemTest86+ is thorough, stand alone memory test for x86 architecture
+systems. BIOS based memory tests are only a quick check and often
+miss failures that are detected by MemTest86+.
+
+%prep
+%autosetup -p1 -n memtest86plus-%{version}
```

Once the archive name is fixed:
```suggestion
%autosetup -p1
```